MAUI/Effects-View/Effects/Selection.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: The .NET MAUI Effects View control | Syncfusion
```

Comment:
The .NET MAUI Selection Effects | Effects View control | Syncfusion

Reply:
Updated.